src/routes/inngang/inngang-innsendingsvalg-post.tsx

```diff
@@ -34,17 +35,18 @@ interface Props {
 
 const InngangInnsendingPost = ({
     temaKey,
-    title = Tema[temaKey],
+    titleKey,
     inngangkategori,
     allowsAnke,
     mailKlageUrl,
     mailAnkeUrl
 }: Props) => {
-    useLogPageView(PageIdentifier.INNGANG_INNSENDING_POST, temaKey, title);
+    useLogPageView(PageIdentifier.INNGANG_INNSENDING_POST, temaKey, titleKey);
+    const title = useTitle(titleKey) ?? 'Tittel mangler';
     usePageInit(`${title} \u2013 klage eller anke`);
     const breadcrumbs: Breadcrumb[] = [
         {
-            title: inngangkategori.title,
```

Comment:
Denne skal vel være `title: inngangkategori.title,` som det stod? Ellers får vi tittel(nøkkelen) to ganger i brødsmulene:

![image](https://user-images.githubusercontent.com/19277989/106612496-7000bc80-6569-11eb-9972-375c988d2bca.png)

(Kan se det på feks https://klage.dev.nav.no/arbeid/arbeidsavklaringspenger )

Reply:
Jupp, ble litt mye "fiksing" der.

Reply:
Fikset!